packages/sdk/client/src/packlets/tests/spaces.test.ts

```diff
@@ -147,4 +150,61 @@ describe('Spaces', () => {
 
     await asyncTimeout(Promise.all([hello.wait(), goodbye.wait()]), 200);
   });
+
+  test('Peer do not load mutations before epoch', async () => {
+    const testBuilder = new TestBuilder();
+
+    const services1 = testBuilder.createLocal();
+    const client1 = new Client({ services: services1 });
+
+    const services2 = testBuilder.createLocal();
+    const client2 = new Client({ services: services2 });
+    await client1.initialize();
+    afterTest(() => client1.destroy());
+    await client2.initialize();
+    afterTest(() => client2.destroy());
+    await client1.halo.createIdentity({ displayName: 'Peer 1' });
+    await client2.halo.createIdentity({ displayName: 'Peer 2' });
+    const space1 = await client1.createSpace();
+    await space1.waitUntilReady();
+
+    const feedKey = services1.host._serviceContext.dataSpaceManager?.spaces.get(space1.key)?.inner.dataFeedKey;
+    const dataSpace1 = services1.host._serviceContext.dataSpaceManager?.spaces.get(space1.key);
+    const feed1 = services1.host._serviceContext.feedStore.getFeed(feedKey!)!;
+
+    const amount = 10;
+    {
+      // Create mutations and epoch.
+      for (const i of range(amount)) {
+        const expando = new Expando({ id: i.toString(), data: i.toString() });
+        space1.db.add(expando);
+      }
+      // Wait to process all mutations.
+      await dataSpace1!.inner.dataPipeline.waitUntilTimeframe(new Timeframe([[feedKey!, amount]]));
+      // Create epoch.
+      await client1.services.services.SpacesService?.createEpoch({ spaceKey: space1.key });
```

Comment:
did this wait for epoch to be processed?

Reply:
Yes